decidim-core/app/helpers/decidim/sanitize_helper.rb

```diff
@@ -35,6 +40,11 @@ def decidim_sanitize_editor(html, options = {})
       content_tag(:div, decidim_sanitize(html, options), class: %w(ql-editor ql-reset-decidim))
     end
 
+    def decidim_sanitize_editor_admin(html, options = {})
+      html = Decidim::IframeDisabler.new(html, options).perform
```

Comment:
We do not have the `Iframe::Disabler` in 0.26.

Please see the following thread for related topic: https://github.com/decidim/decidim/pull/10746#discussion_r1172440474

Reply:
I think we can just remove this line and apply the necessary changes in the specs.

All admin entered iframes are allowed in 0.26 because the data consent management was introduced in 0.27.

Reply:
Done in 471ad43ce62ffd753720eafdd348d00cf1d649ec